ecl/eclagent/eclagent.cpp

```diff
@@ -2502,6 +2502,8 @@ bool EclAgent::checkPersistUptoDate(IRuntimeWorkflowItem & item, const char * lo
         errText.appendf("Rebuilding PERSIST('%s'): Saved CRC isn't present", logicalName);
```

Comment:
Not really sure what happens in this case or how this even occurs but am wondering how this is included in the semantics of REFRESH(false)?